content/docs/openshift/uninstall-kubeflow.md

```diff
@@ -0,0 +1,17 @@
++++
+title = "Uninstall Kubeflow"
+description = "Instructions for uninstalling Kubeflow"
```

Comment:
Suggested revision: add that these instructions apply to OpenShift deployments

Instructions for uninstalling Kubeflow from your OpenShift cluster